ddtrace/tracer/util_test.go

```diff
@@ -36,3 +39,20 @@ func TestToFloat64(t *testing.T) {
 		})
 	}
 }
+
+func TestParseUint64(t *testing.T) {
+	t.Run("negative", func(t *testing.T) {
+		id, err := parseUint64("-8809075535603237910")
+		assert.NoError(t, err)
+		assert.Equal(t, uint64(9637668538106313706), id)
+	})
+	t.Run("positive", func(t *testing.T) {
```

Comment:
Would you mind adding a blank line between these tests? We've done the same everywhere else.